src/Reboot/Reboot.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '../styles';
+import exactProp from '../utils/exactProp';
+
+const styles = theme => ({
+  '@global': {
+    html: {
+      WebkitFontSmoothing: 'antialiased', // Antialiasing.
+      MozOsxFontSmoothing: 'grayscale', // Antialiasing.
+      // Change from `box-sizing: content-box` so that `width`
+      // is not affected by `padding` or `border`.
+      boxSizing: 'border-box',
+    },
+    '*, *::before, *::after': {
+      boxSizing: 'inherit',
+    },
```

Comment:
I'm curious, is there a reason to prefer this approach over simply

```css
*, *::before, *::after {
  box-sizing: border-box;
}
```

?

Reply:
@pelotom This way, people can change the inherited value anywhere. @nathanmarks Introduced this pattern. As far as I remember, Google Map was all broken with `box-sizing: border-box;`. Bootstrap approach was making it harder than necessary to go back `box-sizing: content-box;`.

Reply:
What do you think?

Reply:
Interesting, I didn't know about that, but it makes sense.

Reply:
However, this approach is making it harder to cherry-pick a different box-sizing on a single element.

Reply:
So it's about cherry-picking vs, omg, it doesn't play well with this third party library.

Reply:
I have no idea what's best. 🤔

Reply:
I will benchmark. copying is often the best thing to do. I do it everyday.

Reply:
You could always add a prop which controls which approach you want to use.

Reply:
@pelotom It seems it's Bootstrap vs everybody else.

- https://developer.microsoft.com/en-us/fabric#/components/checkbox
- http://blueprintjs.com/docs/#core/typography
- https://semantic-ui.com/usage/theming.html

We can add a property if people ask for it with a solid use case :).